include/roboteam_ai/stp/Play.hpp

```diff
@@ -5,62 +5,106 @@
 #ifndef RTT_PLAY_HPP
 #define RTT_PLAY_HPP
 
+#include <utilities/Dealer.h>
+
 #include <array>
 
-#include "world_new/World.hpp"
 #include "Role.hpp"
+#include "world_new/World.hpp"
 
 namespace rtt::ai::stp {
 
+/**
+ * Play class that's used in the STP model
+ * on update traverses every Role, and updates it.
+ */
+class Play {
+   public:
+    constexpr static size_t ROBOT_COUNT = 11;
+
+    /**
+     * Initializes tacticInfos vector and calls assignRoles
+     */
+    void initialize() noexcept;
+
+    /**
+     * Updated the stored world
+     * @param world
+     */
+    void updateWorld(world_new::World* world) noexcept;
```

Comment:
You may want to pass by reference

```suggestion
    void updateWorld(world_new::World& world) noexcept;
```